android/src/main/kotlin/com/revenuecat/purchases_flutter/Constants.kt

```diff
@@ -0,0 +1,112 @@
+package com.revenuecat.purchases_flutter
+
+internal object Constants {
+
+    object MethodNameConstants{
```

Comment:
We believe extracting all these constants doesn't provide much value and can even decrease legibility. The exception to this would be when they are repeated. Could you move the constants back to the plugin class? It would be fine to leave any that are repeated.